### ISSUE TITLE
The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible

### ISSUE THREAD

**Issue by jalew188** [MannLabs/alphapeptdeep#194]
```
[.../site-packages/torch/optim/lr_scheduler.py:232]: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
```
